### ISSUE TITLE
Checkboxes in Nested Lists do not Sync

### ISSUE THREAD

**Issue by Nebukadneza** [hackmdio/hackmd#699]
Hey,

Checkboxes are usually synced between view-mode and edit-mode. If i have a note using checkboxes like:
```
- [ ] Project.io
- - [ ] Implement Feature
- - [ ] Do Thing
- Shop in store 1
- - [ ] Buy Thing
[…etc…]
```

In this example, the topmost checkbox will sync its state between view and edit mode. All other checkboxes, however, will not.

I hope that everyone’s okay with considering this as a bug: I believe that _if_ there’s a feature such as checkbox-sync, it should work regardless of _where_ the checkboxes are used.

Thank for your great tool, and Best
-Dario

**Comment by jackycute** [comment]
Hi @Nebukadneza 
Thanks for reporting this.

You found a bug!
And here is the fix: https://github.com/hackmdio/hackmd/pull/700

**Comment by Nebukadneza** [comment]
Wooo, thanks for the quick fix! I’ll test it later to confirm resolution, if that’s alright with you.

**Comment by Nebukadneza** [comment]
I can confirm #700 to fix the problem, thank you a lot! Should i close the bug, or wait for the PR to get merged first?

**Comment by SISheogorath** [comment]
I'll close it when I merge the PR. thanks for your work and thanks to @jackycute for the fix!

**Comment by SISheogorath** [comment]
Fixed by #700
